sass/components/_features.scss

```diff
@@ -0,0 +1,38 @@
+.features {
+  padding: 94px 0 94px;
+
+  &__list {
+    display: flex;
+    gap: 30px;
+    margin: 0;
+    padding: 0;
+    &-item {
```

Comment:
don't use such nesting logic.
`features__list` is one element, while `features__list-item` is another, they should not be nested inside each other